main.py

```diff
@@ -1558,15 +1558,27 @@ def publish(cls, key, *args):
             for listener in cls.listener_dict[key]:
                 listener(*args)
 
+def is_applicable(settings):
+    syntax = settings.get('syntax')
+    return syntax and is_supported_syntax(syntax)
+
+class DefinitionHandler(sublime_plugin.EventListener):
+    def on_text_command(self, view, command_name, args):
+        if not is_applicable(view.settings()):
+            return None
+
+        if command_name != 'context_goto_definition':
+            return None
+
+        return ('lsp_symbol_definition', args)
```

Comment:
Like @tomv564 mentioned we definitely need to check wether the server is active and wether the server supports goto symbol def, otherwise this'll do nothing.